Compatibility/runtimeconfig.template.json

```diff
@@ -0,0 +1,170 @@
+
+    // Switch to enable appending the local assembly version to the Uri being set 
+    // for ResourceDictionary.Source via Baml2006ReaderInternal.
+    "Switch.System.Windows.Baml2006.AppendLocalAssemblyVersionForSourceUri": false,
+
+    // Switch to enable keyboard navigation from a hyperlink to go to the wrong place
+    "Switch.System.Windows.Controls.KeyboardNavigationFromHyperlinkInItemsControlIsNotRelativeToFocusedElement": false,
+
+    // Switch to opt-out of the ItemAutomationPeer weak-reference.
+    // Setting this to true can avoid NRE crashes, but re-introduces some memory leaks
+    "Switch.System.Windows.Automation.Peers.ItemAutomationPeerKeepsItsItemAlive": false,
+
+    // BaseAppContextSwitches
+
+    // Starting .NET 4.6, ExecutionContext tracks Thread.CurrentCulture and Thread.CurrentUICulture, which would be restored
+    // to their respective previous values after a call to ExecutionContext.Run. This behavior is undesirable within the
+    // Dispatcher - various dispatcher operations can run user code that can in turn set Thread.CurrentCulture or
+    // Thread.CurrentUICulture, and we do not want those values to be overwritten with their respective previous values.
+    // To work around the new ExecutionContext behavior, we introduce CulturePreservingExecutionContext for use within
+    // Dispatcher and DispatcherOperation. WPF in .NET 4.6 & 4.6.1 shipped with buggy behavior - each DispatcherOperation
+    // ends with all modifications to culture infos being reverted.Though unlikely, if some applications targeting 4.6 or
+    // above might have taken a dependence on this bug, we provide this compatibility switch that can be enabled by the application.
+    "Switch.MS.Internal.DoNotUseCulturePreservingDispatcherOperations": false,
+
+     //PacakageDigitalSignatureManager.DefaultHashAlgorithm is now SHA256. Setting this flag will make it SHA1 as it
+     //is in legacy scenarios prior to .NET 4.7.1.
+    "Switch.MS.Internal.UseSha1AsDefaultHashAlgorithmForDigitalSignatures": false,
+
+    // Allowing developers to turn off the Invoke added by DDVSO:543980 as there are compat issues with
+    // timing during shutdown for some applications.
+    "Switch.MS.Internal.DoNotInvokeInWeakEventTableShutdownListener": false,
+
+    // Improved the memory allocation and cleanup scheduling behavior of the weak-event pattern
+    "Switch.MS.Internal.EnableCleanupSchedulingImprovements": false,
+
+    // Enable/disable various perf and memory improvements related to WeakEvents
+    "Switch.MS.Internal.EnableWeakEventMemoryImprovements": false,
+
+    // AccessibilitySwitches
+
+    // https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-accessibility-improvements.MD
+    "Switch.UseLegacyAccessibilityFeatures": false,
+
+    // https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-accessibility-improvements-48.md
```

Comment:
This was the 4.7.2 flag, not the 4.8 flag.
This might be a better overall link for the accessibility flags.  It lists all of them and then a set of version-specific changes from 4.7.1+: https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility